components/frontend/src/__fixtures__/fixtures.js

```diff
@@ -79,16 +88,22 @@ export function createTestableSettings() {
         hiddenTags: renderHook(() => useHiddenTagsURLSearchQuery()).result.current,
         metricsToHide: renderHook(() => useMetricsToHideURLSearchQuery()).result.current,
         nrDates: renderHook(() => useNrDatesURLSearchQuery()).result.current,
-        showIssueCreationDate: renderHook(() => useShowIssueCreationDateURLSearchQuery()).result.current,
+        showIssueCreationDate: renderHook(() => useShowIssueCreationDateURLSearchQuery()).result
```

Comment:
Why the line breaks here, max line length? I'd say the previous line was more clear

Reply:
I've solved it by getting rid of the duplication.